server/api.py

```diff
@@ -1637,11 +1637,16 @@ def submit_match_score(
     except Match.DoesNotExist:
         return 400, {"message": "Match does not exist"}
 
-    if match.status in {Match.Status.COMPLETED, Match.Status.YET_TO_FIX}:
+    if (
+        match.status in {Match.Status.COMPLETED, Match.Status.YET_TO_FIX}
+        or match.team_1 is None
+        or match.team_2 is None
+    ):
         return 400, {"message": "Match score cant be submitted in current status"}
 
-    is_authorised, team_id = can_submit_match_score(match, request.user)
```

Comment:
hmm, now that you've replaced the check with `user_tournament_teams`, is the `can_submit_match_score` function being used anywhere else ?

Reply:
Yep its not used anywhere

Reply:
I think we can remove the code?

Reply:
yeah

Reply:
yee will do